File: frontend/src/app/fizio-terapija/fizioTerapijaGetAllResponse.ts
export interface FizioTerapijaGetAllResponse{
  fizioTerapije:FizioTerapijaGetAllResponseFizioTerapija[]
}

export interface FizioTerapijaGetAllResponseFizioTerapija{
fizioTerapijaId:number,
  opis:string,
  datumPostavke:string,
  zaposlenikId:number,
  korisnikDomaID:number
}

File: DomZaStaraLicaApi/Endpoints/FizioTerapija/GetAll/FizioTerapijaGetAllResponse.cs
namespace DomZaStaraLicaApi.Endpoints.FizioTerapija.GetAll
{
    public class FizioTerapijaGetAllResponse
    {
        public List<FizioTerapijaGetAllResponseFizioTerapija> FizioTerapije { get; set;}
    }

    public class FizioTerapijaGetAllResponseFizioTerapija
    {
        public int FizioTerapijaId { get; set; }
        public string Opis { get; set; }
        public DateTime DatumPostavke { get; set; }
        public int ZaposlenikId { get; set; }
        public int KorisnikDomaID { get; set; }
    }
}
